README.md

```diff
@@ -139,6 +139,10 @@ and speed in our CI system.
 - **[MacStadium][23]**: Managed hosting provider for Mac. Provides Mac
   hardware used for testing in the Node.js project's CI system.
 
+- **[Packet][24]**: Bare metal cloud for developers. Through their
+  [works on arm partnership](https://www.worksonarm.com/),
```

Comment:
I think "Works on Arm" is going to be closest to how they are presenting it. Check out https://www.worksonarm.com/blog/woa-issue-23/ and search for "works on arm" and you'll see a couple of instances where it's used mid-sentence and the capitalisation is "Works on Arm". Sorry to be pedantic but other than this LGTM.